code/src/java/pcgen/cdom/content/IndirectCalculation.java

```diff
@@ -94,8 +93,8 @@ public void getDependencies(DependencyManager fdm)
 	}
 
 	@Override
-	public void isValid(FormulaSemantics semantics) throws SemanticsException
-	{
+	public void isValid(FormulaSemantics semantics)
+    {
```

Comment:
looks like the whitespace is out here and in other files?

Reply:
doesn't the checkstyle find that stuff?  weird.  i'll take a look.

Reply:
oh, it's going from tabs to spaces.  what's our preference on this and can we set it up in checkstyle?

also:https://chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom/related

Reply:
@karianna i need a quick clarification on this

Reply:
We have a checkstyle config set up for the project --> https://github.com/PCGen/pcgen/tree/master/code/standards - see what that has in it to start with and if we're missing a rule then we can add one.

Reply:
@karianna there is a rule for spaces.  I found a workaround for enforcing tabs.  what is our preference on tabs or spaces?  my personal preference is spaces because they are a fixed width.

Reply:
anyway. the right thing to do here is to reduce whitespace change and open a  PR with a reformat + checkstyle rule.  i guess i'll open an issue to discuss whitespace after work.  i'll also update this after work.

Reply:
@karianna whitespace fixed and issue created https://github.com/PCGen/pcgen/issues/6298